spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/ChannelBindingService.java

```diff
@@ -82,4 +120,26 @@ public void unbindConsumers(String inputChannelName) {
 	public void unbindProducers(String outputChannelName) {
 		this.binder.unbindProducers(outputChannelName);
 	}
+
+	/**
+	 * Setup data-type and message converters for the given message channel.
+	 *
+	 * @param messageChannel message channel to set the data-type and message converters
+	 * @param channelName the channel name
+	 */
+	public void configureMessageConverters(MessageChannel messageChannel, String channelName) {
+		Assert.isAssignable(AbstractMessageChannel.class, messageChannel.getClass());
```

Comment:
do we need to consider AOP proxies here? (if so we would need to extract the target)